integrations/jenkins/main.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+from loguru import logger
+from enum import StrEnum
+
+from client import JenkinsClient
+from port_ocean.context.ocean import ocean
+
+
+class ObjectKind(StrEnum):
+    JOB = "jenkinsJob"
+    BUILD = "jenkinsBuild"
```

Comment:
please leave the kinds without the prefix

Reply:
changed them back and removed the prefix